junit-plugin/src/main/scala/org/scalajs/junit/plugin/ScalaJSJUnitPlugin.scala

```diff
@@ -388,7 +388,14 @@ class ScalaJSJUnitPlugin(val global: Global) extends NscPlugin {
               varargsModule,
               definitions.wrapVarargsArrayMethodName(definitions.ObjectTpe),
               Nil, List(array))
-          gen.mkMethodCall(definitions.List_apply, List(wrappedArray))
```

Comment:
In this commit's message, maybe change "resist" to "support"? It feels like it's easier to understand.

Reply:
Done. Updated and rebased.